src/App.js

```diff
@@ -8,7 +8,7 @@ const App = () => {
   return (
     <div>
       <Routes />
-      <Text text="It is a text" />
+      <Text color="green" size="medium" text="It is a text" />
```

Comment:
@AnthoniaOkafor, we should be using the correct color specified in the mockup design not green

Reply:
ok @meekg33k